test/file-upload-quirks.test.ts

```diff
@@ -5,6 +5,7 @@ import fs from 'node:fs/promises';
 import DatauriParser from 'datauri/parser';
 import express from 'express';
 import multer from 'multer';
+// @ts-expect-error this file is ESM
```

Comment:
@kanadgupta Is this still needed? 

![Screen Shot 2023-10-04 at 1 41 04 PM](https://github.com/readmeio/fetch-har/assets/33762/c16e5469-cd1a-4fab-bf73-de2097e2ea08)

Reply:
oh whoops yeah as of https://github.com/readmeio/fetch-har/pull/431 we don't need this! fixed in https://github.com/readmeio/fetch-har/commit/0eea0ff5ddd7b25da8c99a26ec4a1df238ea1548